DependencyInjection/Configuration.php

```diff
@@ -361,6 +361,7 @@ public function getConfigTreeBuilder()
 
                                 'bundles/sonataadmin/Admin.js',
                                 'bundles/sonataadmin/treeview.js',
+                                'bundles/sonataadmin/inlineDeleteCheckboxes.js',
```

Comment:
Thought this was to specific (only applies to forms using `sonata_type_collection` with `edit: inline`) to be in Admin.js